dev-tools/README.md

```diff
@@ -1,27 +1,60 @@
-## Setup Instructions
+# Setup Instructions
 
 ```
 # initial setup
 curl "https://raw.githubusercontent.com/AudiusProject/audius-protocol/main/dev-tools/setup.sh" | bash
 
 # refresh terminal for docker
 exit
+```
+
+# Local Dev
+
+Unfortunately `/etc/hosts` does not support wildcards for our service hostname structure.
+For now, update hosts to avoid having to install and run a local dns dependency.
+
+```
+echo "
+127.0.0.1 audius-protocol-discovery-provider-1
+127.0.0.1 audius-protocol-discovery-provider-2
+127.0.0.1 audius-protocol-discovery-provider-3
+127.0.0.1 audius-protocol-creator-node-1
+127.0.0.1 audius-protocol-creator-node-2
+127.0.0.1 audius-protocol-creator-node-3
+127.0.0.1 audius-protocol-creator-node-4
+127.0.0.1 audius-protocol-eth-ganache-1
+127.0.0.1 audius-protocol-poa-ganache-1
+127.0.0.1 audius-protocol-solana-test-validator-1
+127.0.0.1 audius-protocol-identity-service-1
+" | sudo tee -a /etc/hosts
+```
```

Comment:
Half the reason of using docker-compose and sshuttle was to avoid an `/etc/hosts` modification.

This should be resolvable with sshuttle for mapping

Reply:
I will think about this later

Reply:
I do agree w/ cheran here. It feels wrong to have to do this